cko_collection_beta.json

```diff
@@ -1 +1 @@
ategory\": \"Consumer\",\n   \"issuer\": \"GOTHAM STATE BANK\",\n   \"issuer_country\": \"US\",\n   \"product_id\": \"F\",\n   \"product_type\": \"CLASSIC\",\n   \"account_holder\": {\n    \"billing_address\": {\n     \"address_line1\": \"Checkout.com\",\n     \"address_line2\": \"90 Tottenham Court Road\",\n     \"city\": \"London\",\n     \"state\": \"London\",\n     \"zip\": \"W1T 4TJ\",\n     \"country\": \"GB\"\n    },\n    \"phone\": {\n     \"country_code\": \"975\",\n     \"number\": \"174217187\"\n    }\n   }\n  }\n ]\n}","cookie":[],"_postman_previewlanguage":"json"},{"id":"f5127d36-ea69-4670-9953-19f373d95333","name":"Unauthorized","originalRequest":{"url":{"path":["customers",":identifier"],"host":["{{baseUrl}}"],"query":[],"variable":[{"type":"any","key":"identifier"}]},"header":[{"description":{"content":"Added as a part of security scheme: apikey","type":"text/plain"},"key":"Authorization","value":"<API Key>"}],"method":"GET","body":{}},"status":"Unauthorized","code":401,"header":[{"key":"Content-Type","value":"text/plain"}],"body":"","cookie":[],"_postman_previewlanguage":"text"},{"id":"391dcfcb-dbbf-463c-856c-b745eca53070","name":"Customer not found","originalRequest":{"url":{"path":["customers",":identifier"],"host":["{{baseUrl}}"],"query":[],"variable":[{"type":"any","key":"identifier"}]},"header":[{"description":{"content":"Added as a part of security scheme: apikey","type":"text/plain"},"key":"Authorization","value":"<API Key>"}],"method":"GET","body":{}},"status":"Not Found","code":404,"header":[{"key":"Content-Type","value":"text/plain"}],"body":"","cookie":[],"_postman_previewlanguage":"text"}],"event":[]},{"id":"913542f0-db01-4b0e-b3d7-f3473b9a7aac","name":"Update customer details","request":{"name":"Update customer details","description":{"content":"Update details of a customer","type":"text/plain"},"url":{"path":["customers",":identifier"],"host":["{{baseUrl}}"],"query":[],"variable":[{"disabled":false,"type":"any","value":"cus_lKOf_Whw9AV6eRqJlau69OCE0N","key":"identifier","description":"(Required) The customer id"}]},"header":[{"key":"Content-Type","value":"application/json"}],"method":"PATCH","auth":{"type":"apikey","apikey":[{"key":"key","value":"Authorization"},{"key":"value","value":"{{secretKey}}"},{"key":"in","value":"header"}]},"body":{"mode":"raw","raw":"{\n    \"email\": \"JohnTest@test.com\",\n    \"name\": \"John Test\",\n    \"default\": \"src_imu3wifxfvlebpqqq5usjrze6y\",\n    \"phone\": {\n        \"number\": \"4155552671\",\n        \"country_code\": \"+1\"\n    },\n    \"metadata\": {\n        \"coupon_code\": \"NY2018\",\n        \"partner_id\": 123989\n    }\n}"}},"response":[{"id":"92b68b63-6c32-4141-b134-48d351a8f77b","name":"Customer updated successfully","originalRequest":{"url":{"path":["customers",":identifier"],"host":["{{baseUrl}}"],"query":[],"variable":[{"type":"any","key":"identifier"}]},"header":[{"description":{"content":"Added as a part of security scheme: apikey","type":"text/plain"},"
```

Comment:
@ben-ahmady-cko Have you a way of verifying this or are we going off trust in the previous steps?

Reply:
Right now it's in beta so I'm happy with trust, but we can consider ways of verifying it before going GA